code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -603,26 +623,26 @@ nobiliumsuppression = INFINITY
 	var/list/cached_gases = air.gases
 	var/temperature = air.temperature
 	var/old_heat_capacity = air.heat_capacity()
+	var/turf/open/location = isturf(holder) ? holder : null
 	///the more heat you use the higher is this factor
 	var/increase_factor = min((temperature / METAL_HYDROGEN_MINIMUM_HEAT), 5)
 	///the more moles you use and the higher the heat, the higher is the efficiency
 	var/heat_efficency = cached_gases[/datum/gas/hydrogen][MOLES] * 0.01 * increase_factor
 	var/pressure = air.return_pressure()
 	var/energy_used = heat_efficency * METAL_HYDROGEN_FORMATION_ENERGY
-	var/turf/open/location = isturf(holder) ? holder : null
 
 	if(pressure >= METAL_HYDROGEN_MINIMUM_PRESSURE && temperature >= METAL_HYDROGEN_MINIMUM_HEAT)
 		cached_gases[/datum/gas/bz][MOLES] -= heat_efficency * 0.01
 		if (prob(20 * increase_factor))
-			cached_gases[/datum/gas/hydrogen][MOLES] -= heat_efficency * 10
-			if (prob(50 / increase_factor))
+			cached_gases[/datum/gas/hydrogen][MOLES] -= heat_efficency * 3.5
+			if (prob(100 / increase_factor))
```

Comment:
?

Reply:
i found out that metallic hydrogen production was too low and too random to be able to make consistently. That prob(100/increase_factor) will lower itself to prob(20) if the player will just add the heat to the mix without thinking, so lowering the production of the metal. The 3.5 change (from 10) is to not make people waste maybe hours of gameplay to just stay there to produce h2, if when this get merged there will be a surplus of this metal i'll make a PR to balance it out

Reply:
Gottcha